src/components/Reviews.jsx

```diff
@@ -4,27 +4,47 @@ import { Review } from "./Review";
 
 export function Reviews() {
   const [reviews, setReviews] = useState([]);
+  const [categories, setCategories] = useState([]);
+  const [chosenCategory, setChosenCategory] = useState("all");
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
+    let url = "https://be-nc-games.onrender.com/api/reviews";
+    if (chosenCategory !== "all") {
+      url += `?category=${chosenCategory}`;
+    }
+
     axios
-      .get("https://be-nc-games.onrender.com/api/reviews")
+      .get(url)
```

Comment:
Can this be extracted into api.js? Can the axios line on line 23 be a function in api.js that you use instead. Would like all axios requests extracted out please 😁